### ISSUE TITLE
Sponge - Pay Toggle Not Working Properly

### ISSUE THREAD

**Issue by Aiidoneus** [YiC200333/XConomy#61]
**Version of XConomy**  
XConomy-Sponge-2.18.3

**Server version**  
spongeforge-1.12.2-2838-7.4.7

**Storage-type**  
Local

**Describe the issues**  
Hello,
When using the `/pay` it errors out saying "The target user can't receive your payment."
Example command `/pay Aidoneus 100` 

---

**XConomy's config.yml**
https://gist.github.com/Aiidoneus/24f75dd22bf21985cfae883405b91561

**Screenshots**
![](https://i.imgur.com/9gjf5Ub.png)

**Comment by YiC200333** [comment]
does player has 'xconomy.user.pay.receive' permission?

**Comment by Aiidoneus** [comment]
OK, so I did a little more digging around and there seems to be something weird going on with the `/paytoggle` command. Or so I think at least. The above issue only seems to happen if users have the `xconomy.op` permission. 

Last night I gave 2 test users the following permissions;
`xconomy.user.balance`
`xconomy.user.pay`
`xconomy.user.pay.receive`
`xconomy.user.paytoggle` 

When I use the paytoggle command, it says that you reject receiving payment or something similar. I basically get the confirmation that the toggle works but players are still allowed to send money.
